packages/main/src/ComboBox.js

```diff
@@ -490,6 +493,32 @@ class ComboBox extends UI5Element {
 		return !this.readonly;
 	}
 
+	get accInfo() {
+		return {
+			"role": "combobox",
```

Comment:
Are these overwritten from somewhere (maybe multicombobox?) If yes, it's fine, if not -please hardcode them in the template. No need for variable parts for static content.

Reply:
You are right. We discussed to remove it.

Reply:
Done.